src/Header/HeaderWrap.php

```diff
@@ -90,7 +90,7 @@ protected static function wrapStructuredHeader($value, StructuredInterface $head
      */
     public static function mimeEncodeValue($value, $encoding, $lineLength = 998)
     {
-        return Mime::encodeQuotedPrintableHeader($value, $encoding, $lineLength, Headers::EOL);
+        return Mime::encodeBase64Header($value, $encoding, $lineLength, Headers::EOL);
```

Comment:
This is a rather big change. Shouldn't users be able to opt-in between quoted-printable and base64?